packages/components/inputs/selectable-search-input/src/selectable-search-input.spec.tsx

```diff
@@ -134,6 +134,35 @@ describe('SelectableSearchInput', () => {
     ).toHaveTextContent('Bar');
   });
 
+  it('should show the passed experimental value when controlled externally', () => {
```

Comment:
Thanks for working in these new tests.

I think we should also test that, when the consumer state changes, the internal `SelectableSearchInput` component state also change when using the new property but does not when using the old one.

Reply:
Sorry, maybe I do not understand this clearly but it seems like that is what the first test checked for.

Reply:
My bad.
Thanks for explaining in a meeting.